.eslintrc

```diff
@@ -6,5 +6,31 @@
   },
   "globals": {
     "__DEV__": true
+  },
+  "plugins": [
+    "es5"
+  ],
+  "rules": {
+    "es5/no-es6-methods": "error",
+    "es5/no-es6-static-methods": "error",
+    "es5/no-binary-and-octal-literals": "error",
+    "es5/no-classes": "error",
+    "es5/no-for-of": "error",
+    "es5/no-generators": "error",
+    "es5/no-object-super": "error",
+    "es5/no-typeof-symbol": "error",
+    "es5/no-unicode-code-point-escape": "error",
+    "es5/no-unicode-regex": "error",
+    "es5/no-computed-properties": "off",
+    "es5/no-destructuring": "off",
+    "es5/no-default-parameters": "off",
+    "es5/no-spread": "off",
+    "es5/no-modules": "off",
+    "es5/no-exponentiation-operator": "off",
+    "es5/no-block-scoping": "off",
+    "es5/no-arrow-functions": "off",
+    "es5/no-shorthand-properties": "off",
+    "es5/no-rest-parameters": "off",
+    "es5/no-template-literals": "off"
```

Comment:
Suggestions on how to better organize this plugin are welcome! I could use the presets they define, but not sure I want to hide the rules they have right now

Reply:
I can agree that having the rules listed here would be much more clear for when we reconsider IE11 support in the future.

Are the rules that are disabled ones that do get transpiled by Babel?

Reply:
I've read through Babel docs and when no `target` is specified in `.babelrc` it automatically targets es2015+ transpiling every new syntax it finds!
The caveat here is that in every new syntax it finds, bloatware is add on final build (the magic transforming spread in something usable in ie11)
For new methods in Array/Obj/String, babel delegates the polyfills to another library, core-js. It's possible to improve the babelrc to only add the needed polyfills from core-js (not the whole library), if someday a new method is really needed!
That said, in this list of rules, the most important for us (which breaks in IE11 without polyfills) are the:
- "es5/no-es6-methods": "error",
- "es5/no-es6-static-methods": "error",
- "es5/no-generators": "error",

The rest of rules flagged as errors, are new syntax which generates bloatware by babel. The disabled ones are the ones used across this repo, and the active ones are the ones not being used in this repo (preventing those syntaxes to be used)